packages/app-store/webex/DESCRIPTION.md

```diff
@@ -0,0 +1,8 @@
+---
+items:
+  - 1.jpeg
```

Comment:
@aar2dee2 can you add proper images and logo for the app.

Reply:
Sure. Do you use a specific app to get the screenshots?

Reply:
@aar2dee2 I haven't added screenshots by myself yet.  Simple screen grabs should be enough depicting  what the app can achieve,

Reply:
added